README.md

```diff
@@ -1 +1,25 @@
-# mparticle-android-integration-rokomobi
\ No newline at end of file
+## Instabot Kit Integration
+
+[See here for more information](https://github.com/mParticle/mparticle-android-sdk/wiki/Kit-Development) on how to use this example to write a new kit.
+
+This repository contains the [Instabot](https://instabot.io/) integration for the [mParticle Android SDK](https://github.com/mParticle/mparticle-android-sdk).
+
+### Adding the integration
+
+1. Add the kit dependency to your app's build.gradle:
+
+    ```groovy
+    dependencies {
+        compile 'com.mparticle:android-instabot-kit:4+'
+    }
+    ```
+2. Follow the mParticle Android SDK [quick-start](https://github.com/mParticle/mparticle-android-sdk), then rebuild and launch your app, and verify that you see `"Instabot detected"` in the output of `adb logcat`.
+3. Reference mParticle's integration docs below to enable the integration.
+
+### Documentation
+
+[Example integration](http://docs.mparticle.com/?java#REPLACEME)
```

Comment:
place replace the link text and the url with `http://docs.mparticle.com/integrations/instabot`